models/index.js

```diff
@@ -16,6 +16,26 @@ var env       = process.env.NODE_ENV || 'development';
 var config    = require(__dirname + '/../config/config.js')[env];
 var db        = {};
 
+function UserException(message) {
+  this.message = message;
+  this.name = 'UserException';
+}
+
+if (typeof env == 'undefined'){
+  throw new UserException('env is undefined')
+} else if (env != 'development' || env != 'test' || env != 'production') {
```

Comment:
This failed on travis, maybe because we aren't using the right equality test - pretty much all the time in javascript you have to use `===` or `!==` because javascript is bad

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness